openmc/deplete/msr.py

```diff
@@ -0,0 +1,190 @@
+            Depletable material id to where the element gets transferred
+
+        """
+        mat = self._get_mat_id(mat)
+        check_value('Element', element, ELEMENT_SYMBOL.values())
+        if element in self.removal_rates[mat]:
+            return self.removal_rates[mat][element][1]
+
+    def get_elements(self, mat):
+        """Extract removing elements for a given material
+
+        Parameters
+        ----------
+        mat : Openmc.Material or str or int
+            Depletable material
+
+        Returns
+        -------
+        elements : list
+            List of elements where removal rates exist
+
+        """
+        mat = self._get_mat_id(mat)
+        if mat in self.removal_rates.keys():
+            return self.removal_rates[mat].keys()
+
+    def set_removal_rate(self, mat, elements, removal_rate, removal_rate_units='1/s',
+                         destination_material=None):
+        """Set removal rate to elements in a depletable material.
+
+        Parameters
+        ----------
+        mat : openmc.Material or str or int
+            Depletable material
+        elements : list of str
+            List of strings of elements that share removal rate
+        removal_rate : float
+            Removal rate
+        destination_material : Openmc.Material or str or int, Optional
+            Destination material to where nuclides get fed.
+        removal_rate_units: {'1/s', '1/min', '1/h', '1/d', '1/a'}
+            Units for values specified in the removal_rate argument. 's' means
+            seconds, 'min' means minutes, 'h' means hours, 'a' means Julian years.
+
+        """
+        mat = self._get_mat_id(mat)
+        check_type('removal_rate', removal_rate, Real)
+
+        if destination_material is not None:
+            destination_material = self._get_mat_id(destination_material)
+            if len(self.burn_mats) > 1:
+                check_value('destination_material', str(destination_material),
+                            self.burn_mats)
+            else:
+                raise ValueError(f'Transfer to material {destination_material} '\
+                        'is set, but there is only one depletable material')
+
+        if removal_rate_units in ('1/s', '1/sec'):
+    	    unit_conv = 1
+        elif removal_rate_units in ('1/min', '1/minute'):
+    	    unit_conv = _SECONDS_PER_MINUTE
+        elif removal_rate_units in ('1/h', '1/hr', '1/hour'):
+    	    unit_conv = _SECONDS_PER_HOUR
+        elif removal_rate_units in ('1/d', '1/day'):
+    	    unit_conv = _SECONDS_PER_DAY
+        elif removal_rate_units in ('1/a', '1/year'):
+    	    unit_conv = _SECONDS_PER_JULIAN_YEAR
+        else:
+    	    raise ValueError("Invalid removal rate unit '{}'".format(removal_rate_units))
+
+        for element in elements:
+            check_value('Element', element, ELEMENT_SYMBOL.values())
+            self.removal_rates[mat][element] = removal_rate / unit_conv, destination_material
```

Comment:
Why is there a comma between `unit_conv` and `desination_material`? Are there two values to set here?